cpp/src/arrow/adapters/arrow-rados-cls/scripts/splitter.py

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think this is more of a "skyhook" utility and less of a "Arrow utility".  It maybe shouldn't be included in this PR.

Reply:
Fixed